crates/bevy_app/src/app.rs

```diff
@@ -714,12 +711,12 @@ impl App {
     ///         a + b
     ///     })
     ///     // Registering an existing function with its type name
-    ///     .register_function_with_name(std::any::type_name_of_val(&mul), mul)
+    ///     .register_function_with_name(core::any::type_name_of_val(&mul), mul)
```

Comment:
Documentation has also been updated where appropriate due to the linting requirements. This is visible to users of Bevy, but I think a reasonable change.